addons/finalAlert/README.md

```diff
@@ -0,0 +1,40 @@
+# finalAlert
+Displays Final Fantasy 7 style alerts when enemies use skills or magic. Allows you to emphasize skills by name (useful for interrupts).
+
+## Commands:
+
+### Test window
```

Comment:
I'm honestly not sure if this command should be even present in the finished addon, feel even less enthused about it being documented. But up to you. It seems like a debug command and those should not be user facing imo.